Dragonflight/WarriorArms.lua

```diff
@@ -699,12 +749,19 @@ spec:RegisterAbilities( {
         texture = 236303,
         range = 8,
 
+        spend = -20,
```

Comment:
Is Bladestorm generating 20 Rage on use, or does it generate during the channel?

Reply:
PTR is 0 rage (generating with damage), BETA is 20 flat rage on use.  Will change to 0 and add it back to the RegisterResource mimicking SL's rage generation.
PTR:
![image](https://user-images.githubusercontent.com/5103286/197064624-08e727b9-9a13-429d-8030-07442fb44382.png)
BETA:
![image](https://user-images.githubusercontent.com/5103286/197064734-a894af9b-cd79-4fd3-843c-38696b78f55c.png)

Reply:
If it flat out gives you 20 on use, this is fine.